lib/app/modules/teamPage/views/team_page_view.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:managment_app/Themes/themes.dart';
+import '../../../../components/teamTile.dart';
+import '../controllers/team_page_controller.dart'; // Import the TeamTile component
+
+class TeamPageView extends GetView<TeamPageController> {
+  const TeamPageView({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      // Background gradient image
+      body: Container(
+        decoration: const BoxDecoration(
+          image: DecorationImage(
+            image: AssetImage('assets/images/bg.image.gradient.png'),
+            fit: BoxFit.cover,
+          ),
```

Comment:
use svg image and  instead of taking whole background as svg image give a background color and with the help of stack widget place that vector at bottom right